packages/core/src/components/dialog/dialog.tsx

```diff
@@ -80,6 +80,11 @@ export interface IDialogProps extends OverlayableProps, IBackdropProps, Props {
      */
     transitionName?: string;
 
+    /**
+     * Ref supplied to the `Classes.DIALOG_CONTAINER` element.
+     */
+    dialogRef?: IRef<HTMLDivElement>;
```

Comment:
Let's name this containerRef:

```suggestion
    containerRef?: IRef<HTMLDivElement>;
```